src/prism-live.js

```diff
@@ -365,6 +365,11 @@ var _ = Prism.Live = class PrismLive {
 				this.wrapper.style[prop] = cs[prop];
 				this.textarea.style[prop] = this.pre.style[prop] = "inherit";
 			}
+			console.log(prop)
```

Comment:
Whoops :)

Reply:
Yikes, my bad. This is fixed by commit #25e5949